corehq/motech/repeaters/models.py

```diff
@@ -439,23 +438,36 @@ def _time_request(self, repeat_record, payload, timing_context):
             return self.send_request(repeat_record, payload)
 
     def fire_for_record(self, repeat_record, timing_context=None):
+        def is_success(resp):
```

Comment:
How would you feel about moving this to the bottom of the module, alongside `format_response()` and `is_response()`? The benefit is that it would be testable.

Reply:
Yeah that sounds good. I still plan to do that refactor, but notably omitted those specific changes from my first followup PR as I just want to get the improved logging change out safely.